test/Microsoft.DocAsCode.MarkdigEngine.Tests/ChromelessFormsTest.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.MarkdigEngine.Tests
+{
+	using Microsoft.DocAsCode.Common;
+	using Microsoft.DocAsCode.Plugins;
+	using Xunit;
+	using System.Linq;
+
+	public class ChromelessFormsTest
+	{
+		static public string LoggerPhase = "ChromelessForms";
+
+		[Fact]
+		public void ChromelessFormsTestWithoutModel()
+		{
+			var content = @"::: form action=""create-resource"" submitText=""Create"" :::";
+			var expected = @"<form class=""chromeless-form"" data-action=""create-resource"">
+<button type=""submit"">Create</button>
+</form>
+".Replace("\r\n", "\n");
+
+			TestUtility.AssertEqual(expected, content, TestUtility.MarkupWithoutSourceInfo);
+		}
+
+		[Fact]
+		public void ChromelessFormsTestWithModel()
+		{
+			var content = @"::: form model=""./devsandbox/ChromelessFormsTest.md"" action=""create-resource"" submitText=""Do it"" :::";
+			var expected = @"<form class=""chromeless-form"" data-model=""./devsandbox/ChromelessFormsTest.md"" data-action=""create-resource"">
+<fieldset disabled=""disabled"">
+<div></div>
+<button type=""submit"">Do it</button>
+</fieldset>
+</form>
+".Replace("\r\n", "\n");
+
+			TestUtility.AssertEqual(expected, content, TestUtility.MarkupWithoutSourceInfo);
+		}
+
+		[Fact]
+		public void ChromelessFormsAttributeValueRequired()
+		{
+			var content = @"::: form submitText :::";
+			var listener = TestLoggerListener.CreateLoggerListenerWithPhaseEqualFilter(LoggerPhase);
+
+			Logger.RegisterListener(listener);
+			using (new LoggerPhaseScope(LoggerPhase))
+			{
+				TestUtility.MarkupWithoutSourceInfo(content);
+			}
+			Logger.UnregisterListener(listener);
+
+			// Listener should have an error message and not output.
+			Assert.NotEmpty(listener.Items.Where(x => x.Code == "invalid-form"));
+		}
+
+		[Fact]
+		public void ChromelessFormsTestActionRequired()
```

Comment:
add a test case where `submitText` is `"`

Reply:
Added. Nothing breaks, although in practice I believe the content authors would have to escape the `"` for it to register properly.

Reply:
Sorry, I was trying to say `>` and see if it is properly escaped: `<button type=\"submit\">{submitText}</button>` --> `<button type=\"submit\">{HtmlUtility.HtmlEncode(submitText)}</button>`

Reply:
Ah, good catch! I will add this.

Reply:
The `submitText` variable is now encoded using System.Net.WebUtility, and there is a test for the output.